src/map/LayerHelper.js

```diff
@@ -296,17 +303,25 @@ exports.prototype.getFlatLayers = function(layer) {
  * of others groups.
  * @param {ol.layer.Base} layer The base layer, mostly a group of layers.
  * @param {Array.<ol.layer.Base>} array An array to add layers.
+ * @param {number|undefined} computedOpacity Opacity inherited from ancestor layer groups.
  * @return {Array.<ol.layer.Layer>} Layers.
  * @private
  */
-exports.prototype.getFlatLayers_ = function(layer, array) {
-  if (layer instanceof olLayerGroup) {
+exports.prototype.getFlatLayers_ = function(layer, array, computedOpacity) {
```

Comment:
Can you use an optional arguments for "computedOpacity" instead of calling this function with "undefined" at line 297 ?

Reply:
After discussion with @gberaudo, we prefer that it is explicit that the value is called as 'undefined' instead of skipping the last argument.

Reply:
Okay